ProcessMaker/Models/Script.php

```diff
@@ -318,13 +347,43 @@ private function setDefaultExecutor()
         }
     }
 
-    private function timeout()
+    /**
+     * This method retrieves the error number by type.
+     * In case of not finding it, -1 is returned.
+     */
+    private function getErrorHandlingNumberByType($type)
     {
-        $errorHandlingTimeout = Arr::get($this->errorHandling, 'timeout', null);
-        if (is_numeric($errorHandlingTimeout)) {
-            return (int) $errorHandlingTimeout;
+        $result = Arr::get($this->errorHandling, $type, null);
+        if (is_numeric($result)) {
+            return (int) $result;
         }
+        return -1;
+    }
 
-        return $this->timeout;
+    /**
+     * Get the value of the timeout property if there hasn't been a defined error in the errorHandling array.
+     */
+    private function timeout()
+    {
+        $result = $this->getErrorHandlingNumberByType('timeout');
+        return $result === -1 ? $this->timeout : $result;
+    }
+
+    /**
+     * Get the value of the retryAttempts property if there hasn't been a defined error in the errorHandling array.
+     */
+    private function retryAttempts()
+    {
+        $result = $this->getErrorHandlingNumberByType('retryAttempts');
```

Comment:
This should be retry_attempts to the value set by ErrorHandlingRetryAttempts.vue